### PR TITLE
duckduckhack.com

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/duckduckhack.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+www.duckduckhack.com mismatch
+docs.duckduckhack.com mismatch
```
Comment (J0WI) [inline src/chrome/content/rules/duckduckhack.com.xml]: This works for me now

File: src/chrome/content/rules/duckduckhack.com.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+-->
+<ruleset name="duckduckhack.com">
+	<target host="duckduckhack.com" />
+	<target host="forum.duckduckhack.com" />
+	
+	<rule from="^http://www\.duckduckhack\.com/"
```
Comment (J0WI) [inline src/chrome/content/rules/duckduckhack.com.xml]: www needs to be added as target host then.

**Comment by J0WI** [comment]
Please also add a reference in/to the DuckDuckGo ruleset.

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/duckduckhack.com.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+-->
+<ruleset name="duckduckhack.com">
+	<target host="duckduckhack.com" />
+	<target host="docs.duckduckhack.com" />
+	<target host="forum.duckduckhack.com" />
+
+	<target host="www.duckduckhack.com" />
+
+	<rule from="^http://duckduckhack\.com/"
+		to="https://www.duckduckhack.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/duckduckhack.com.xml]: Now you redirect both to the mismatching www

Comment (galeksandrp) [inline src/chrome/content/rules/duckduckhack.com.xml]: Fixed.

File: src/chrome/content/rules/duckduckhack.com.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/duckduckhack.com.xml]: Please link to DuckDuckGo.xml here

Comment (galeksandrp) [inline src/chrome/content/rules/duckduckhack.com.xml]: Added.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**